docs/source/docker/overview.rst

```diff
@@ -9,12 +9,14 @@ and an easy way to work with lightly. But there is more!
     :alt: Alt text
     :figclass: align-center
 
-With the introduction of our on-premise solution, you can process larger datasets completely on your end without data leaving your infrastructure.
+With the introduction of our on-premise solution, you can **process larger datasets completely on your end without data leaving your infrastructure**.
 We worked hard to make this happen and are very proud to present you with the following specs:
 
-* **NEW** See your docker runs live in the Lightly Platform (see :ref:`ref-docker-runs` )
+* **NEW** Automatically upload the sampled dataset to the Lightly Platform (see :ref:`ref-docker-upload-to-platform`)
```

Comment:
This does not really fit:
> without data leaving your infrastructure
vs.
>Automatically upload the sampled dataset to the Lightly Platform

Reply:
Why does it not fit? Could you suggest an alternative?

Reply:
If the sampled dataset is automatically uploaded to the Lightly Platform, then data is leaving the infrastructure of our customers. This made me stumble. 

Alternative:
**NEW** The on-premise docker can now be used easily together with the Lightly Platform for more control and insights. Your docker runs and their current progress can be seen on the Platform. Furthermore, you can set the sampled dataset to be uploaded the Lightly Platform to view browse the chosen images and embeddings. This allows you to better understand your dataset, e.g. to find out where data is missing.